_src/custom-date-format.md

```diff
@@ -34,11 +34,11 @@ mod my_date_format {
     //
     // although it may also be generic over the input types T.
     pub fn serialize<S>(
-        date: &DateTime<Utc>,
+        date: &NaiveDateTime,
         serializer: S,
     ) -> Result<S::Ok, S::Error>
-    where
-        S: Serializer,
+        where
+            S: Serializer,
```

Comment:
Please keep the original formatting. It is the formatting applied by rustfmt.

Reply:
I did a run of rustfmt, I hope it is okay now.

Reply:
Was that with some particular rustfmt.toml configuration you have locally? Rustfmt with default settings should not be indenting the where-clauses.

Reply:
Indeed, sorry for that. Will be fixed when I adressed your other comment.

Reply:
Updated in the most recent commit